Description: Provide path to asmtools.jar
 A number of hotspot tests use java asmtools. Provide path to asmtools.jar
Author: Vladimir Petko <vladimir.petko@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/jtreg6/+bug/2015011
Last-Update: 2023-04-03
--- a/make/build.xml
+++ b/make/build.xml
@@ -144,7 +144,7 @@
             <manifest>
                 <attribute name="Built-By" value="${user.name}"/>
                 <attribute name="Main-Class" value="com.sun.javatest.regtest.Main"/>
-                <attribute name="Class-Path" value="javatest.jar jh.jar junit.jar"/>
+                <attribute name="Class-Path" value="javatest.jar jh.jar junit.jar asmtools.jar"/>
                 <attribute name="jtreg-Name" value="jtreg"/>
                 <attribute name="jtreg-Version" value="${build.version}"/>
                 <attribute name="jtreg-Milestone" value="${build.milestone}"/>
